hapi/hedera-protobufs/services/state/blockstream/block_stream_info.proto

```diff
@@ -85,7 +85,7 @@ message BlockStreamInfo {
      * The latest available hash SHALL be for block N-1.<br/>
      * This is REQUIRED to implement the EVM `BLOCKHASH` opcode.
      * <p>
-     * <h5>Field Length</h5>
+     * <b>Field Length</b>
```

Comment:
This is not correct.  We'll need to figure out how to make javadoc relax about the header tag, as a simple bold doesn't produce nearly the correct output in markdown (which is what these are for; we shouldn't even be running javadoc on the PBJ generated files).

Reply:
Javadoc, very incorrectly, requires headings be _nested_ and in order (starting from 2).
This can be replaced with the following for now (yes, it's a bit ugly, but so is javadoc's standard doclet):
```suggestion
     * <div style="display: block;font-size: .83em;margin-top: 1.67em;margin-bottom: 1.67em;margin-left: 0;margin-right: 0;font-weight: bold;">Field Length</div>
```

Reply:
Awesome. 💯